doc/sphinx/addendum/generalized-rewriting.rst

```diff
@@ -209,7 +209,7 @@ They also support the :attr:`universes(polymorphic)` attributes.
    the relation is not an equivalence relation. The proofs must be
    instances of the corresponding relation definitions: e.g. the proof of
    reflexivity must have a type convertible to
-   :g:`reflexive (A t1 … tn) (Aeq t′ 1 … t′ n)`.
+   :g:`reflexive (A t1 … tn) (Aeq t′1 … t′m)`.
```

Comment:
m or n?  How do you know it should be m?  (t'1 suggests it's related to t1, so I'd expect the same number of primed items.)